xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -8,132 +8,146 @@
     </para>
 <screen>cd ~/scratch/ansible/next/ardana/ansible
-grep -A1 FND-MDB--first-member hosts/verb_hosts</screen>
+grep -A1 FND-MDB--first-member hosts/verb_hosts
+[FND-MDB--first-member:children]
+ardana002-cp1-c1-m1</screen>
+   </step>
+   <step xml:id="mariadb-nodes">
     <para>
-     Example output:
+     Find the host IP address which will be used to log in.
     </para>
-<screen>$ grep -A1 FND-MDB--first-member hosts/verb_hosts
-[FND-MDB--first-member:children]
-ardana-cp1-c1-m1</screen>
-   </listitem>
-   <listitem>
+<screen>cat /etc/hosts | grep ardana002-cp1-c1-m1
+10.84.43.82      ardana002-cp1-c1-m1-extapi ardana002-cp1-c1-m1-extapi
+192.168.24.21    ardana002-cp1-c1-m1-mgmt ardana002-cp1-c1-m1-mgmt
+10.1.2.1         ardana002-cp1-c1-m1-guest ardana002-cp1-c1-m1-guest
+10.84.65.3       ardana002-cp1-c1-m1-EXTERNAL-VM ardana002-cp1-c1-m1-external-vm
+    </screen>
     <para>
-     Log in to the first host member from the <literal>FND-MDB</literal> group.
+     In this example, <literal>192.168.24.21</literal> is the IP address for
+     the host.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Become root:
+     SSH into the host.
     </para>
-<screen>sudo su</screen>
-   </listitem>
-   <listitem>
+<screen>ssh ardana@192.168.24.21</screen>
+   </step>
+   <step>
     <para>
-     Source the backup environment file
+     Source the backup file.
     </para>
-<screen>source /home/stack/backup.osrc</screen>
-   </listitem>
-   <listitem>
+<screen>source /var/lib/ardana/backup.osrc</screen>
+   </step>
+   <step>
+    <para>
+     Find the client id.
+    </para>
+<screen>
+<?dbsuse-fo font-size="0.65em"?>
+freezer client-list
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| Client ID                   | uuid                             | hostname                    | description |
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| ardana002-cp1-comp0001-mgmt | f4d9cfe0725145fb91aaf95c80831dd6 | ardana002-cp1-comp0001-mgmt |             |
+| ardana002-cp1-comp0002-mgmt | 55c93eb7d609467a8287f175a2275219 | ardana002-cp1-comp0002-mgmt |             |
+| ardana002-cp1-c0-m1-mgmt    | 50d26318e81a408e97d1b6639b9404b2 | ardana002-cp1-c0-m1-mgmt    |             |
+| ardana002-cp1-c1-m1-mgmt    | 78fe921473914bf6a802ad360c09d35b | ardana002-cp1-c1-m1-mgmt    |             |
+| ardana002-cp1-c1-m2-mgmt    | b2e9a4305c4b4272acf044e3f89d327f | ardana002-cp1-c1-m2-mgmt    |             |
+| ardana002-cp1-c1-m3-mgmt    | a3ceb80b8212425687dd11a92c8bc48e | ardana002-cp1-c1-m3-mgmt    |             |
++-----------------------------+----------------------------------+-----------------------------+-------------+</screen>
+    <para>
+     In this example, the client id of host
+     <literal>ardana002-cp1-c1-m1-mgmt</literal> is
+     <literal>ardana002-cp1-c1-m1-mgmt</literal>.
```

Comment:
This is probably not the best example, because hostname and client ID are identical. Can we use an example where this is not the case? On the other hand, if name and ID usually _are_ the same, we should keep this example. In that case, the sentence should be rephrased:

          <para>
          In this example, the <literal>hostname</literal> and the <literal>Client ID</literal>
          are the same: <literal>ardana002-cp1-c1-m1-mgmt</literal>.
          </para>

Reply:
I tried to find the example where it is different, but I could not. thinks as long as ask user to pick up the  client id corresponding to the hostname for example ardana002-cp1-c1-m1-mgmt, it should be ok.